src/styles/navigation_section.css

```diff
@@ -90,6 +96,10 @@
     padding: 0 .5em;
 }
 
+[role=banner] .coz-nav-group:not(.coz-nav--error).coz-nav-group--grey {
```

Comment:
Is `--grey` the  best semantic specifier ? Maybe something like `--inactive` should be better ? What do you think ?